project/Dependencies.scala

```diff
@@ -3,30 +3,32 @@ import sbt.Keys._
 
 object Dependencies {
 
-  private[this] def onScala213(onScala213: String, onOthers: String) = Def.setting {
-    CrossVersion.partialVersion(scalaVersion.value) match {
-      case Some((2, 13)) => onScala213
-      case _ => onOthers
+  private[this] def onScala213(onScala213: String, onOthers: String) =
+    Def.setting {
```

Comment:
Since you can't remove this utility method, can you make sure the style remains the same as before? Same for all the other reverted changes.

Reply:
Done :)